lib/main.dart

```diff
@@ -21,8 +13,9 @@ class MyApp extends StatelessWidget {
       theme: ThemeData(
         primarySwatch: Colors.blue,
       ),
-      home:UserHomePage(),
+      home:ChatBot(),
```

Comment:
This is changed for testing purpose only it should be replaced by the original UserHomePage() after testing